tests/appsec/api_security/test_schemas.py

```diff
@@ -186,11 +186,11 @@ def test_request_method(self):
             schema,
             [
                 {
-                    "main[0][key]": ANY,
-                    "main[0][value]": ANY,
-                    "main[1][key]": ANY,
-                    "main[1][value]": ANY,
-                    "nullable": ANY,
+                    "main[0][key]": [8],
+                    "main[0][value]": [8],
+                    "main[1][key]": [8],
+                    "main[1][value]": [8],
+                    "nullable": [8],
```

Comment:
@cbeauchesne I changed it because ANY will match even when the key is missing. So before this change, the test is actually not testing anything :/

Reply:
I'd rather have you fix the ANY thingy

Reply:
cc @christophe-papazian who wrote the tests. I'm currently trying to understand the logic

Reply:
Fixed in master :+1: